code_samples/action_basics/build.gradle

```diff
@@ -3,7 +3,6 @@ plugins {
   id 'org.jetbrains.intellij' version '0.4.7'
```

Comment:
0.4.9

Reply:
Updated. I have to see if there is a way to say "use the latest production version of this plugin"